Create AutomateServiceRequestCreationBasedOnCounter.vb

sybethanlevy opened the pull request:
Creation of example breakpoint based on CF https://partner.sybiz.com/Support/SupportDetail/125437

sybethanlevy committed:
Create AutomateServiceRequestCreationBasedOnCounter.vb

Creation of example breakpoint based on CF https://partner.sybiz.com/Support/SupportDetail/125437

Marjon80 commented:
Do you get a service request number to appear in your message?

John Kennison
Software Development Manager (Mon,Tues,Thurs,Fri)
My working day may not be your working day. Please don’t feel obliged to reply to this e-mail outside of your normal working hours.

[Sybiz Software]<https://www.sybiz.com/>
For the latest updates from Sybiz, click here<https://www.sybiz.com/latest-news>


From: sybethanlevy ***@***.***>
Sent: Friday, 12 May 2023 4:43 PM
To: Sybiz/Vision ***@***.***>
Cc: John Kennison ***@***.***>; Assign ***@***.***>
Subject: [Sybiz/Vision] Create AutomateServiceRequestCreationBasedOnCounter.vb (PR #8)


Creation of example breakpoint based on CF https://partner.sybiz.com/Support/SupportDetail/125437

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Sybiz/Vision/pull/8

Commit Summary

  *   978d668<https://github.com/Sybiz/Vision/pull/8/commits/978d6688a199afc273696d4afe169b7d82a69948> Create AutomateServiceRequestCreationBasedOnCounter.vb

File Changes

(1 file<https://github.com/Sybiz/Vision/pull/8/files>)

  *   A BreakpointSamples/Implementing Custom Business Rules and Logic/AutomateServiceRequestCreationBasedOnCounter.vb<https://github.com/Sybiz/Vision/pull/8/files#diff-d921289becfd603927891a23cc9431f7d3f7aeda5a02ec0902d5dcab09ec3a5a> (36)

Patch Links:

  *   https://github.com/Sybiz/Vision/pull/8.patch
  *   https://github.com/Sybiz/Vision/pull/8.diff

—
Reply to this email directly, view it on GitHub<https://github.com/Sybiz/Vision/pull/8>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO4FASRBUWO5D4F7OLI6UQLXFXPGDANCNFSM6AAAAAAX7DW5AY>.
You are receiving this because you were assigned.Message ID: ***@***.***>

sybethanlevy committed:
Update AutomateServiceRequestCreationBasedOnCounter.vb

- No I don't get a service request number, I just get Automatic. I originally had a different mechanism there and forgot to properly retrofit when I switched gears
- Added "greater than or equals' to line 14 as opposed to equals.

sybethanlevy committed:
Update AutomateServiceRequestCreationBasedOnCounter.vb

It's almost 5pm on a Friday I don't want to talk about it :(
